code/modules/vehicles/motorized_wheelchair.dm

```diff
@@ -150,28 +154,20 @@
 		explosion(src, -1, 1, 3, 2, 0)
 		visible_message("<span class='boldwarning'>[src] explodes!!</span>")
 		return
-	if(t5 >= 15 && A.density && isliving(A)) //FULP: If T5 is greater than or equal to 15, run that fucker over and just keep on driving!
-		var/mob/living/D = A
-		var/atom/throw_target = get_edge_target_turf(D, pick(GLOB.cardinals))
-		D.throw_at(throw_target, 2, 3) //FULP: YEET
-		D.Knockdown(85)
-		D.adjustStaminaLoss(40) //FULP: Just like a mulebot, this does no damage on the initial hit- but it does LOADS of damage if you run a prone person over (see Crossed() in carbon.dm for the code for that))
-		D.visible_message("<span class='danger'>[src] rams into [D], sending [D] flying!</span>", "<span class='userdanger'>[src] rams you!</span>")
-		return
-	// FULP: If T5 (T5 identifier) is less than 15, execute as normal
+	///// FULP-ONLY - The Great T5 Maintenance - Removed the snowflake code that basically prevented the person riding a quantum wheelchair from being knocked off. This will nerf its offensive abilities by a lot. - GoldenAlpharex - 09/12/2020
```

Comment:
you should mention this in the opening post and the changelog

Reply:
Good idea. I don't know why I only received the notification about this today.